ansible/provision-private-proxied-cluster-airgap-rpm.yml

```diff
@@ -0,0 +1,57 @@
+### This is not a truely airgapped cluster!!! The intent here is specifically to test the install flow for an airgapped install
```

Comment:
It's unclear whether this playbook is a test, an example, or something our clients can use. If it's an example, can we move it to another directory? `examples/` perhaps.

Reply:
Not sure what happened but my comments I made by phone last week all appear to have been gobbled 🤦‍♀️ anyway, the intent here is to satisfy two different needs:
* the need for a testable playbook for airgap features
* the need for an example for users to show them how we can use the airgap features

I've included the comment to indicate that this isn't actually an airgapped cluster, but I don't have a good idea how to communicate the rest of that just yet.